automatic/screencloud/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,17 @@
+$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$fileName = 'ScreenCloud-0.0-x86.msi'
+
+$packageArgs = @{
+    packageName    = 'screencloud'
+    softwareName   = 'ScreenCloud'
+
+    fileType       = 'msi'
+    file           = "$toolsDir\$fileName"
+
+    silentArgs     = '/qn /norestart'
+    validExitCodes = @(0)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
```

Comment:
Add `rm "$toolsDir\$fileName" -ea 0` after installation as installer is no longer needed and takes up considerable space.